patches/net/minecraft/server/ReloadableServerResources.java.patch

```diff
@@ -34,9 +43,20 @@
      public static CompletableFuture<ReloadableServerResources> loadResources(
          ResourceManager p_248588_,
          LayeredRegistryAccess<RegistryLayer> p_335667_,
-@@ -90,14 +_,27 @@
+@@ -82,31 +_,62 @@
+         Commands.CommandSelection p_249301_,
+         int p_251126_,
+         Executor p_249136_,
+-        Executor p_249601_
++        Executor p_249601_,
++        java.util.Set<net.neoforged.neoforge.flag.Flag> enabledModdedFlags
```

Comment:
IMO this should be the flag manager, and should also be passed after the FeatureFlagSet.